sql/_13_issues/_23_1h/cases/cbrd_24843_2.sql

```diff
@@ -0,0 +1,647 @@
+/* ----------------------------------------
+ *
+ * test case 36:
+ *
+ * 1. 'c.p_a = 1' -> pt_lambda_node() -> 'p.a = 1' [o]
+ * 2. 'p.a = 2' -> pt_lambda_node() -> 'c.p_a = 2' [o]
+ * 3. 'p.a = 3 and p.b = 3' -> pt_lambda_node() -> 'c.p_a = 3 and c.p_b = 3' [o]
+ * 4. 'p.a > 3 and p.a = 4' -> pt_lambda_node() -> 'c.p_a = 4' [o]
+ * 5. '(c.p_a = 5 or c.p_b = 5)' [o]
+ * 6. '(p.a = 6 or p.b = 6)' [x]
+ * 7. 'p.a = cast (7 as numeric (1, 0))' [o]
+ * 8. 'p.a >= 8' [x]
+ * 9. 'p.a in (1, 3, 5, 7, 9)' [x]
+ * 10. 'p.a between 2 and 4' [x]
+ * 11. 'p.a = (select a from t_parent order by a desc limit 1)' [x]
+ * 12. 'c.p_a <> 1' -> pt_lambda_node() -> 'p.a <> 1' [o]
+ *     'c.p_a != 1' -> pt_lambda_node() -> 'p.a != 1' [o]
+ * 13. 'c.p_a like 1' -> pt_lambda_node() -> 'p.a like 1' [o]
+ * 14. 'p.a = 4' -> pt_lambda_node() -> 'c.p_a = 4' group by c.b [o]
+ * 15. 'p.a = 4' -> pt_lambda_node() -> 'c.p_a = 4' group by c.b having count(*) > 2 [o]
```

Comment:
I don't know why this comment was added.
Cases added by the development team and cases added by the QA team should be separated into files.

This comment is incorrect.
pt_lambda_node() does not behave like this.